networkx/utils/backends.py

```diff
@@ -971,25 +971,18 @@ def _call_if_no_backends_installed(self, /, *args, backend=None, **kwargs):
         """Returns the result of the original function (no backends installed)."""
         if backend is not None and backend != "networkx":
             raise ImportError(f"'{backend}' backend is not installed")
+        if "networkx" not in self.backends:
+            raise NotImplementedError(
+                f"`{self.name}' is not implemented by 'networkx' backend. "
+                "This function is included in NetworkX as an API to dispatch to "
+                "other backends."
+            )
```

Comment:
Adding this code block is optional, and is useful to have a single place to control the error message. We could also call the original networkx function, which should also raise `NotImplementedError`.